experimental/Schemes/Types.jl

```diff
@@ -259,10 +259,19 @@ the list ``f₁,…,fᵣ`` as the *generators* for ``U``.
       end
     end
     U = new{SpecType, typeof(base_ring(X))}(X, f)
-    U.intersections = Dict{Tuple{Int, Int}, SpecType}()
+    U.intersections = Dict{Tuple{Int, Int}, AbsSpec}()
```

Comment:
Why this change?

Reply:
Again, because we need the flexibility and this was a deprecated leftover from older code.